### PR TITLE
[GUI][Zerocoin] transaction list - handle basecoin address decoding

### PR DESCRIPTION
**Pull request by codeofalltrades**
Line 414 is cleanup from code reviews. 
Line 512 was missed with the original fix

### CODE REVIEW

File: src/qt/transactionrecord.cpp [hunk 1]
```diff
@@ -532,7 +530,7 @@ QList<TransactionRecord> TransactionRecord::decomposeTransaction(const interface
             //
             parts.append(TransactionRecord(hash, nTime, nSize, TransactionRecord::Other, "", nNet, 0));
             parts.last().involvesWatchAddress = involvesWatchAddress;
-        }
+        }
```
Comment (Zannick) [inline src/qt/transactionrecord.cpp:533]: nit: drop the extra space?

**Review by Zannick — APPROVED** [review]
utACK ab801ee9cb92438e3ea354e91ad85c7bed440979

**Review by CaveSpectre11 — APPROVED** [review]
utACK https://github.com/Veil-Project/veil/pull/953/commits/ab801ee9cb92438e3ea354e91ad85c7bed440979

**Review by WetOne — APPROVED** [review]

**Comment by WetOne** [comment]
utACK ab801ee9cb92438e3ea354e91ad85c7bed440979

**Comment by codeofalltrades** [comment]
This pr doesn't fix the issue. I was able to recreate the issue on Win10.

**Pull request closed without merging**